lib/retrieve-tag.js

```diff
@@ -2,7 +2,7 @@ const {spawn} = require('@npmcli/git')
 const semver = require('semver')
 
 module.exports = async opts => {
-  const tag = (await spawn(['describe', '--abbrev=0'], opts)).stdout.trim()
+  const tag = (await spawn(['describe', '', '--abbrev=0'], opts)).stdout.trim()
```

Comment:
Did you mean for this to be an empty string?  The discussion makes it seem like this second arg should be `'--tags'`, no?

Reply:
Yikes, oops. I used GitHub's web IDE to make this change, and I did not notice that made the commit with the wrong content.

I just force-pushed the correct code.